src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java

```diff
@@ -38,7 +38,11 @@ public FileAnalysisReport analyzeFits(FileAnalysisReport inputReport,
         String code = params.getOrDefault("product_type", "");
         String level = params.getOrDefault("processing_level", "").toUpperCase();
         String inst = params.getOrDefault("instrument", "").toUpperCase();
-        inputReport.setAdditionalImageParams(Collections.singletonMap("CUBE_FIRST_FRAME","50%"));
+        String configuration = params.getOrDefault("configuration", "").toUpperCase();
+
+        if(! inst.equals("FORCAST") && ! configuration.equals("IMAGING")) {
```

Comment:
I don’t know exactly which the condition is but it seems this is open for error. There must be many things that is not forcast and not imaging.  If this is what you want then ok.

Reply:
Ya, for now those two conditions that I need to catch, everything else should go through the "set cube to middle" process. This is how it was before this ticket.